_board/xinabox_cc03.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: download
+board_id: "xinabox_cc03"
+title: "CC03 download"
+name: "XinaBox CC03"
+manufacturer: "XinaBox"
+board_url: "https://drive.google.com/file/d/13xfg0SRZEeO-_xMElqdf0Sne8rjAcfyn/view"
+board_image: "xinabox_cc03.jpg"
+features:
+  - ARM Cortex-M0+ Architecture
```

Comment:
Please see the guide page at https://learn.adafruit.com/how-to-add-a-new-board-to-the-circuitpython-org-website/adding-to-downloads. On features, please only use the items in that list. If those are not features, either just leave it empty or remove the features block altogether. Thanks.

Reply:
Thanks @makermelissa, I've updated .mds and uploaded new images.